src/main/java/org/folio/circulation/resources/CirculationTenantAPI.java

```diff
@@ -0,0 +1,63 @@
+package org.folio.circulation.resources;
+
+import static org.folio.circulation.support.http.server.JsonHttpResponse.created;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.folio.circulation.support.RouteRegistration;
+import org.folio.circulation.support.http.server.ServerErrorResponse;
+import org.folio.rest.util.OkapiConnectionParams;
+import org.folio.util.pubsub.PubSubClientUtils;
+
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+
+public class CirculationTenantAPI {
+  private static final Logger logger = LoggerFactory.getLogger(CirculationTenantAPI.class);
+
+  public void register(Router router) {
+    RouteRegistration routeRegistration = new RouteRegistration("/_/tenant", router);
+    routeRegistration.create(this::postTenant);
+  }
+
+  public void postTenant(RoutingContext routingContext) {
```

Comment:
It handles `POST /_/tenant`, I believe Okapi calls it when the module is enabled for the tenant. Also, this class now handles `DELETE /_/tenant`. Probably they should be named something like `enableModuleForTenant` and `disableModuleForTenant` respectively?

Reply:
> Probably they should be named something like `enableModuleForTenant` and `disableModuleForTenant` respectively?

I'm happy with explicit names like that. I'm ambivalent about the distinction between activate or enable.

Reply:
Okapi docs refer to this as "enable module for tenant", so I renamed the methods accordingly. It's still a bit misleading though, because we're not enabling or disabling module here, Okapi does.

Reply:
> Okapi docs refer to this as "enable module for tenant", so I renamed the methods accordingly. It's still a bit misleading though, because we're not enabling or disabling module here, Okapi does.

Agreed, it is misleading or confusing, from Okapi's perspective, the module is being enabled for a tenant, whereas from the modules perspective it is being asked to prepare itself for the tenant.